templates/blink/intent_to_implement.html

```diff
@@ -204,6 +204,64 @@ <h4>Contact emails</h4>
    {% endif %}
 {% endif %}
 
+
+<br><br><h4>Estimated milestones</h4>
+<table>
+
+  {% if feature.dt_milestone_desktop_start %}
+    <tr><td>DevTrial on desktop</td>
+    <td>{{feature.dt_milestone_desktop_start}}</td></tr>
+  {% endif %}
+
+  {% if feature.dt_milestone_andorid_start %}
+    <tr><td>DevTrial on android</td>
+    <td>{{feature.dt_milestone_android_start}}</td></tr>
+  {% endif %}
+
+  {% if feature.dt_milestone_ios_start %}
+    <tr><td>DevTrial on iOS</td>
+    <td>{{feature.dt_milestone_ios_start}}</td></tr>
+  {% endif %}
+
+  {% if feature.dt_milestone_webview_start %}
+    <tr><td>DevTrial on Webview</td>
+    <td>{{feature.dt_milestone_webview_start}}</td></tr>
+  {% endif %}
+
+  {% if feature.ot_milestone_desktop_start %}
+    <tr><td>OriginTrial first</td>
+    <td>{{feature.ot_milestone_desktop_start}}</td></tr>
+  {% endif %}
+
+  {% if feature.ot_milestone_desktop_end %}
```

Comment:
Why no origin trial fields for Android and WebView?

Reply:
I thought we might be removing ot_milestone_android_start and _end.  However, now it seems like we just need to explain exactly what they mean.  I have added them here.